Chinese/斗战胜佛/斗战胜佛.md

```diff
@@ -1,4 +1,4 @@
 孙悟空取经回来后，就变成了斗战胜佛！他从一个叛逆者，历经千辛万苦终于成了一个体制内的人，可是沿途的妖怪那么轻松就成仙成佛了，这个世界跟对人十分重要啊！
-
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.